.firebaserc

```diff
@@ -1,5 +0,0 @@
-{
-  "projects": {
-    "default": 'codelabz-dev'
-  }
-}
```

Comment:
Hi, dont remove the code from the existing files.

Reply:
added the file:)